pkg/network/usm/ebpf_ssl.go

```diff
@@ -525,6 +538,38 @@ func (o *sslProgram) ConfigureOptions(options *manager.Options) {
 
 // PreStart is called before the start of the provided eBPF manager.
 func (o *sslProgram) PreStart() error {
+	sslCtxByPIDTGIDMapObj, _, err := o.ebpfManager.GetMap(sslCtxByPIDTGIDMap)
```

Comment:
consider adding a simple function to remove the overhead here (get the map, check its error, create map cleaner, check its error)